extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -668,62 +805,82 @@ public function searchSupportsCurrentLanguage() {
 
 	/**
 	 * Determines if a given language code is supported for dynamic search
+	 *
 	 * @param string $languageCode
+	 *
 	 * @return boolean
 	 */
 	public function searchSupportsLanguageCode( $languageCode ) {
 		return in_array( $languageCode, $this->getGlobal( 'WikiaSearchSupportedLanguages' ) );
 	}
 
 	public function getThumbnailUrl( $pageId, $dimensions = null ) {
-		$width = (isset( $dimensions[ 'width' ] ) ) ? $dimensions[ 'width' ] : static::THUMB_DEFAULT_WIDTH;
-		$height = (isset( $dimensions[ 'height' ] ) ) ? $dimensions[ 'height' ] : static::THUMB_DEFAULT_HEIGHT;
+		$width = ( isset( $dimensions['width'] ) ) ? $dimensions['width'] : static::THUMB_DEFAULT_WIDTH;
+		$height = ( isset( $dimensions['height'] ) ) ? $dimensions['height'] : static::THUMB_DEFAULT_HEIGHT;
 		$imgSource = $this->getImageServing( $pageId, $width, $height );
 		//get one image only
 		$img = $imgSource->getImages( 1 );
 		if ( !empty( $img ) ) {
-			return $img[ $pageId ][ 0 ][ 'url' ];
+			return $img[$pageId][0]['url'];
 		}
+
 		return false;
 	}
 
 	/**
 	 * Gets image serving for page, moved to external method for easier testing
+	 *
 	 * @param $pageId
 	 * @param $width
 	 * @param $height
+	 *
 	 * @return \ImageServing
 	 */
 	protected function getImageServing( $pageId, $width, $height ) {
-		return new \ImageServing( array( $pageId ), $width, $height );
+		return new \ImageServing( [ $pageId ], $width, $height );
 	}
 
-	public function getThumbnailHtml( $pageId, $transformParams = null, /* WikiaGrid 1 column width */ $htmlParams = array( 'desc-link'=>true, 'img-class'=>'thumbimage', 'duration'=>true ) ) {
+	public function getThumbnailHtml(
+		$pageId,
+		$transformParams = null, /* WikiaGrid 1 column width */
+		$htmlParams = [ 'desc-link' => true, 'img-class' => 'thumbimage', 'duration' => true ]
+	) {
 		$html = '';
 		$img = $this->getFileForPageId( $pageId );
-		if ( ! empty( $img ) ) {
-			$transformParams[ 'width' ] = (isset( $transformParams[ 'width' ] ) ) ? $transformParams[ 'width' ] : static::THUMB_DEFAULT_WIDTH;
-			$transformParams[ 'height' ] = (isset( $transformParams[ 'height' ] ) ) ? $transformParams[ 'height' ] : static::THUMB_DEFAULT_HEIGHT;
+		if ( !empty( $img ) ) {
+			$transformParams['width'] =
+				( isset( $transformParams['width'] ) ) ? $transformParams['width'] : static::THUMB_DEFAULT_WIDTH;
+			$transformParams['height'] =
+				( isset( $transformParams['height'] ) ) ? $transformParams['height'] : static::THUMB_DEFAULT_HEIGHT;
 			$thumb = $img->transform( $transformParams );
 			$html = $thumb->toHtml( $htmlParams );
 		}
+
 		return $html;
 	}
 
 	/**
 	 * @param string $pageTitle
 	 * @param array|null $transformParams
+	 *
 	 * @return string|null - html of thumbnail with play button
 	 */
 	public function getThumbnailHtmlFromFileTitle( $pageTitle, $transformParams = null ) {
 		$file = null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$file" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)